src/routes/console/(migration-wizard)/resource-form.svelte

```diff
@@ -0,0 +1,375 @@
+                    return data;
+                });
+            } else if (checked) {
+                formData.update((data) => {
+                    data[parent].root = checked;
+                    return data;
+                });
+            }
+        };
+    }
+
+    function deselectAll() {
+        $formData = deepMap($formData, (v) => {
+            if (typeof v === 'boolean') {
+                return false;
+            }
+            return v;
+        });
+    }
+
+    function selectAll() {
+        $formData = resourcesToMigrationForm(resources, version);
+    }
+
+    let report: any;
+    $: version = report?.version || '0.0.0';
+
+    let isOpen = false;
+    onMount(async () => {
+        isOpen = true;
+        try {
+            switch ($provider.provider) {
+                case 'appwrite': {
+                    const res = await sdk.forProject.migrations.getAppwriteReport(
+                        providerResources.appwrite,
+                        $provider.endpoint,
+                        $provider.projectID,
+                        $provider.apiKey
+                    );
+                    report = res;
+                    break;
+                }
+                case 'supabase': {
+                    const res = await sdk.forProject.migrations.getSupabaseReport(
+                        providerResources.supabase,
+                        $provider.endpoint,
+                        $provider.apiKey,
+                        $provider.host,
+                        $provider.username,
+                        $provider.password,
+                        $provider.port
+                    );
+                    report = res;
+                    break;
+                }
+                case 'firebase': {
+                    const res = await sdk.forProject.migrations.getFirebaseReport(
+                        providerResources.firebase,
+                        $provider.serviceAccount
+                    );
+                    report = res;
+                    break;
+                }
+                case 'nhost': {
+                    const res = await sdk.forProject.migrations.getNHostReport(
+                        providerResources.nhost,
+                        $provider.subdomain,
+                        $provider.region,
+                        $provider.adminSecret,
+                        $provider.database,
+                        $provider.username,
+                        $provider.password
+                    );
+                    report = res;
+                }
+            }
+        } catch (e) {
+            if (!isOpen) return;
+            addNotification({
+                message: e.message,
+                type: 'error'
+            });
+        }
+
+        return () => {
+            isOpen = false;
+        };
+    });
+
+    $: console.log(report);
+    $: resources = providerResources[$provider.provider];
+
+    // $: wizard.setNextDisabled(!report);
```

Comment:
Should this be removed or uncommented?

Reply:
Uncommented. @PineappleIOnic Are all reports working now?